airflow/api_connexion/schemas/variable_schema.py

```diff
@@ -23,7 +23,7 @@ class VariableSchema(Schema):
 
     key = fields.String(required=True)
     value = fields.String(attribute="val", required=True)
-
+    description = fields.String(attribute="description", required=True)
```

Comment:
I don't think ``description`` should be a required field

Reply:
thank you @ephraimbuddy for review & suggestion. 
Will update the changes for 'required' attribute & re-submit the PR.